tests/integration/src/test/java/org/apache/pulsar/tests/integration/proxy/TestProxy.java

```diff
@@ -53,6 +65,7 @@
         proxyViaURL = new ProxyContainer(clusterName, "proxy-via-url")
             .withEnv("brokerServiceURL", "pulsar://pulsar-broker-0:6650")
             .withEnv("brokerWebServiceURL", "http://pulsar-broker-0:8080")
+            .withEnv("webSocketServiceEnabled", "true")
```

Comment:
IIUC if we always enable websockets in this test we are losing to test the case in which websocket is disabled
is it correct ?
it would be better to have tests that cover default configurations together with tests that test additional features

Reply:
Make sense, I have updated the test.